crates/rattler_conda_types/src/run_export.rs

```diff
@@ -5,14 +5,7 @@ use serde::{Deserialize, Serialize};
 pub enum RunExportKind {
```

Comment:
Could you also add documentation here about what they mean? Or refer to the docs of the other struct?